tests/_helpers.py

```
@@ -0,0 +1,37 @@
+            strategy.dim = dim_value
+
+
+def build_3d_engine(dim_value: Dim = Dim.THREE) -> SimulationEngine:
+    """A ``SimulationEngine(is_3d=True)`` whose strategies are patched to ``dim_value``.
+
+    Used by tests that need to exercise the 3D code path before real ``Dim.THREE``
+    resolvers exist for every event type.
+
+    TODO: This is a temporary measure. Drop the dim patching once every resolver
+    slot has a ``Dim.THREE``-capable strategy (cushion + ball-ball; tracked by
+    issues #308–#312). At that point this helper can be replaced by a direct
```

Ruff (0.15.13)

[warning] Docstring contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF002)